features/step_definitions/filtering_steps.rb

```diff
@@ -526,3 +526,16 @@
     end
   end
 end
+
+And(/^I press (tab) key to navigate$/) do |key|
+  find_field('Search').send_keys key.to_sym
+end
+
+Then(/^I should (see|not see) a "Skip to results" link$/) do |can_be_seen|
+  visibility = can_be_seen === 'see'
```

Comment:
this should be an `==` - pesky javascript :-D

Reply:
🙄 thats what happens when you've just been coding html,css & js for the past 3 months.

Well spotted @sihugh 👍